src/main/resources/alfresco/heartbeat/heartbeat-context.xml

```diff
@@ -16,10 +16,14 @@
     <bean id="hbDataCollectorService" class="org.alfresco.heartbeat.HBDataCollectorServiceImpl">
         <constructor-arg value="${heartbeat.enabled}" />
         <property name="hbDataSenderService" ref="hbDataSenderService" />
+        <property name="scheduler" ref="schedulerFactory" />
+        <property name="jobLockService" ref="jobLockService" />
+        <property name="licenseService" ref="licenseService" />

```

Comment:
Remove <property name="licenseService" ref="licenseService" /> 
This will result in bean not found exception